.travis.yml

```diff
@@ -1,4 +1,6 @@
 # Documentation: http://docs.travis-ci.com/user/languages/julia/
+# for some reason, Linux builds are failing when we are at 50 commits, https://github.com/travis-ci/travis-ci/issues/4942
+# adding these lines to get commit 51
```

Comment:
apparently https://github.com/travis-ci/travis-ci/issues/4942

They had the exact same issue. I didn't notice it was the 50th commit until I read that thread. 

I'm still not sure if it is true, but rerunning the builds didn't fix it but 1 extra commit did, so it might just be the strangest bug I've seen yet.